Kickstarter-iOS/Library/DeprecatedWebViewController.swift

```diff
@@ -85,17 +85,17 @@ internal class DeprecatedWebViewController: UIViewController {
 
 extension DeprecatedWebViewController: UIWebViewDelegate {
   // Call super from subclasses that override this method.
-  internal func webViewDidStartLoad(webView: UIWebView) {
+  internal func webViewDidStartLoad(_ webView: UIWebView) {
     self.viewModel.inputs.webViewDidStartLoad()
   }
 
   // Call super from subclasses that override this method.
-  internal func webViewDidFinishLoad(webView: UIWebView) {
+  internal func webViewDidFinishLoad(_ webView: UIWebView) {
     self.viewModel.inputs.webViewDidFinishLoad()
   }
 
   // Call super from subclasses that override this method.
-  internal func webView(webView: UIWebView, didFailLoadWithError error: NSError) {
-    self.viewModel.inputs.webViewDidFail(withError: error)
+  internal func webView(_ webView: UIWebView, didFailLoadWithError error: Error) {
+    self.viewModel.inputs.webViewDidFail(withError: error as NSError?)
```

Comment:
i'll be getting rid of most `NSError`s. it's weird stuff the swift migrator inserted...